packages/core/utils/src/dml/helpers/create-mikro-orm-entity.ts

```diff
@@ -165,6 +189,7 @@ function defineBelongsToRelationship(
       mapToPk: true,
       fieldName: camelToSnakeCase(`${relationship.name}Id`),
       nullable: relationship.nullable,
+      onDelete: shouldCascade ? "cascade" : undefined,
```

Comment:
I think the default case should be to `set null`, right? Otherwise you Postgres won't allow you to delete the data (maybe that's how it is already?), since it will reference a non-existent foreign key

Reply:
When I looked at our relationships in modules, I either saw `onDelete: "cascade"` or nothing. So not sure if we should define `set null`. Have to verify once

Reply:
@adrien2p Any insights on this?